include/devicetree.h

```diff
@@ -42,6 +42,7 @@
  * _ENUM_IDX: property's value as an index into bindings enum
  * _ENUM_TOKEN: property's value as a token into bindings enum (string
  *              enum values are identifiers)
+ * _STRING_TOKEN: string property's value as a token
```

Comment:
Please keep this list in alphabetical order by inserting this at the end, after the `_NAME` suffixes.

Reply:
ok